src/cement_ai_platform/data/processors/bigquery_schema_design_e7775640-39d0-453c-9099-9aa7ef51ab9a.py

```
@@ -0,0 +1,179 @@
+    print(f"  Partition: {schema_info['partition_field']} ({schema_info['partition_type']})")
+    print(f"  Clustering: {', '.join(schema_info['clustering_fields'])}")
+    print(f"  Columns: {len(schema_info['schema'])}")
+    print()
+
+# Create DDL statements for BigQuery
+def generate_ddl_statements(schemas, dataset_name="cement_plant_data"):
+    """Generate BigQuery DDL statements"""
+
+    ddl_statements = {}
+
+    for table_key, schema_info in schemas.items():
```

Ruff (0.13.1)

Loop control variable `table_key` not used within loop body

Rename unused `table_key` to `_table_key`

(B007)